### ISSUE TITLE
QT threads error on 3d_ldm_tutorial.py

### ISSUE THREAD

**Issue by IanHOBD** [Project-MONAI/GenerativeModels#505]
I'm running 3d_ldm_tutorial.py and this code:

`check_data = first(train_loader)
idx = 0

img = check_data["image"][idx, 0]
fig, axs = plt.subplots(nrows=1, ncols=3)
for ax in axs:
    ax.axis("off")
ax = axs[0]
ax.imshow(img[..., img.shape[2] // 2], cmap="gray")
ax = axs[1]
ax.imshow(img[:, img.shape[1] // 2, ...], cmap="gray")
ax = axs[2]
ax.imshow(img[img.shape[0] // 2, ...], cmap="gray")`

Causes the following problem with qt:

QObject::moveToThread: Current thread (0x596fff9c53d0) is not the object's thread (0x596fffa53cd0).
Cannot move to target thread (0x596fff9c53d0)

Any idea why?  It doesn't look like it should be running on a different thread.  Only thing I can think of is that the data loader worker gets check_data owned by a different thread, but I presume this code worked for the author.

**Comment by ericspod** [comment]
It's possible this is a Windows issue or whatever host you're running this script in, since it doesn't use Qt itself. We have this as a notebook now after porting everything into MONAI Core: https://github.com/Project-MONAI/tutorials/blob/main/generation/3d_ldm/3d_ldm_tutorial.ipynb Please try this tutorial and see if you can reproduce your issue with the current version of MONAI and not using GenerativeModels. If you have the issue still you can post a new issue on the Tutorials repo and not this one which is now going to be read-only soon.

**Comment by IanHOBD** [comment]
I'm using python3.10 on Ubuntu 22 by the way. 

I shall try the core version, thanks.
